src/app/getting-started-guide/getting-started-guide.component.html

```diff
@@ -0,0 +1,4 @@
+ This is getting started guide. We can go through the below link for the introduction of Rxjs.
+<div>
+  <a (click)='goToUrl()' [routerLink]="">Introduction to Reactive programming by Andre Staltz</a>
```

Comment:
This should like to the "getting started" guide in the current docs.

Reply:
Can you make this a plain `href` with target `_blank`? I think it would be better to open in new tab rather than leaving docs for external links.